min-max/maxMin.js

```diff
@@ -0,0 +1,42 @@
+var arr = [5,10,45,100,80,7,9,108,1,500];
+
+var value = getMaxOrMin(arr,'min');
+console.log(value);
+
+
+function getMaxOrMin(arr,option) {
+    var maxMin = option.toUpperCase();
```

Comment:
You can remove this line, take option.toUpperCase() in switch

Reply:
ok, got it.